lib/report/generateReport.ts

```diff
@@ -201,18 +211,77 @@ export class CoverageView {
           runtimeExceptions: element.runtimeExceptions,
         });
       });
+
+      const generalErrorsInnerOrigin = this.validationResultsForRendering.filter((x) => {
+        return x.errors && x.errors.length > 0;
+      });
+
       this.coverageResults.forEach((element) => {
         const specLink = this.overrideLinkInReport
           ? `${this.specLinkPrefix}/${element.spec?.substring(
               element.spec?.indexOf("specification")
             )}`
           : `${element.spec}`;
+
+        let errorOperationIds = generalErrorsInnerOrigin.map(
+          (item) => item.operationInfo?.operationId
+        );
+        let passOperations: ValidationPassOperationsFormatInner[] = element.coveredOperationsList
+          .filter((item) => errorOperationIds.indexOf(item.operationId) === -1)
+          .map((item) => {
+            return {
+              key: item.operationId.split("_")[0],
+              operationId: item.operationId,
+            };
+          });
+
+        const passOperationsInnerList: ValidationPassOperationsFormatInner[][] = Object.values(
```

Comment:
I don't see anything that jumps out at me in the code block. Once I get a bit more clarity I'll have a better understanding though.

Reply:
This code is similar with this[ existing code](https://github.com/Azure/oav/blob/d24383eb554abd7fa656f566d630f8702d49babf/lib/swaggerValidator/trafficValidator.ts#L319), which is used to group passed operations by APIs.